src/containers/treePage/ducks/selectors.ts

```diff
@@ -42,7 +42,7 @@ export const getLatestEntry = (
   if (asyncRequestIsComplete(items)) {
     return Object.entries(items.result.entries[0]).reduce<Entry[]>(
       (soFar, [key, value]) => {
-        if (SiteEntryNames[key] && value !== null) {
+        if (SiteEntryNames[key] && (value || value === false)) {
```

Comment:
❓ We don't want to show an empty box where there aren't any site notes?

Reply:
Personally it didn't make sense to me since it felt like something was missing.